cryptoservice/crypto_service_test.go

```diff
@@ -0,0 +1,64 @@
+package cryptoservice
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+
+	"github.com/docker/notary/trustmanager"
+	"github.com/endophage/gotuf/data"
+	"github.com/endophage/gotuf/signed"
+)
+
+func TestED25519(t *testing.T) {
```

Comment:
If you're testing create and sign you should also probably test Get and Delete? Also, we don't use the word "Delete" anywhere. I think we always use "Remove" instead.